rdt-open-reader/src/main/java/io/ona/rdt/util/Constants.java

```diff
@@ -148,4 +150,8 @@
     interface Preference {
         String LOCATION_TREE = "location_tree";
     }
+
+    interface RESULT_CODE {
+        int PERMISSION_FAILED = Activity.RESULT_FIRST_USER + 1;
```

Comment:
Cool, could we rename this to `PERMISSIONS_DENIED`?

Reply:
done